package.json

```diff
@@ -17,26 +22,31 @@
   },
   "dependencies": {
     "dot": "~1.0.3",
-    "svgo": "~0.5.1",
-    "canvas": "~1.1.2",
+    "svgo": "~0.6.1",
+    "canvas": "~1.3.5",
```

Comment:
That breaks upon `npm install` on my machine (Linux node v5.4.1 npm v3.3.12) with some nan error `‘class v8::String’ has no member named ‘GetExternalAsciiStringResource’`.

Historically, upgrading canvas has been a phenonculously difficult task. I recommend skipping that.